libs/components/modals/src/lib/modules/modal/modal-content.component.ts

```diff
@@ -8,5 +12,11 @@ import { Component, ViewEncapsulation } from '@angular/core';
   templateUrl: './modal-content.component.html',
   styleUrls: ['./modal-content.component.scss'],
   encapsulation: ViewEncapsulation.None,
+  providers: [
+    {
+      provide: SkyMediaQueryService,
+      useExisting: SkyResizeObserverMediaQueryService,
+    },
+  ],
```

Comment:
I removed this provider and the demo still worked. Is this needed since we're providing `SkyMediaQueryService` in the host service?

Reply:
This was redundant to what `modal-host.component.ts` is providing and is not needed.